src/main/resources/reference.conf

```diff
@@ -27,6 +27,9 @@ scorex {
     # Enable/disable CORS support
     corsAllowed = false
 
+    # Allowed origin line for cors header value
+    corsAllowedOrigin = "*"
```

Comment:
Instead of having `corsAllowed: Boolean` and `corsAllowedOrigin: String`, we could have just a single `corsAllowedOrigin: Option[String]`.  When it is `None`, cors is not allowed. When it is `Some(s)`, cors is allowed with origin `s`.